### PR TITLE
Fixed the usage information in documentation of "save_screenshot".

### LINKED ISSUE #3651: Typo in documentation for python api for "save_screenshot" method. Want to fix it.
I found a typo here: https://seleniumhq.github.io/selenium/docs/api/py/webdriver_remote/selenium.webdriver.remote.webdriver.html#selenium.webdriver.remote.webdriver.WebDriver.save_screenshot

The method is `save_screenshot` and but it's usage is using `get_screenshot_as_file` as can be seen here
screen shot 2017-03-14 at 2 46 33 pm

I wanted to fix it by modifying the docs for `save_screenshot` method but `save_screenshot` is just an alias to `get_screenshot_as_file`. Therefore I could not change the docs for it.

How can I fix this typo?

Thanks

### PR DESCRIPTION
**Pull request by igauravsehrawat**
"save_screenshot" method was assigned from "get_screenshot_as_file"
as result of that it's usage information was
"driver.get_screenshot_as_file('/Screenshots/foo.png')" which is wrong.
So I defined a separate method which is calling
"get_screenshot_as_file" but with it's correct usage information as
"driver.save_screenshot('/Screenshots/foo.png')"

Fixes #3651

- [x] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

**Comment by igauravsehrawat** [comment]
As per code contributor's guidelines. I ran the tests using `.go clean test` but it didn't pass due some `Could not start a new session.` message, it seems like system configuration problem. If anyone can guide me what I am missing that would great.

Verbose error message: https://gist.github.com/igauravsehrawat/fc66e8c8822b6c2fa61f31127372f525

**Comment by Dude-X** [comment]
@igauravsehrawat Why is aliasing get_screenshot_as_file _wrong_? It may not be the cleanest code, but I don't see a reason to duplicate code.

**Comment by igauravsehrawat** [comment]
@Dude-X Thanks for replying.

@cgoldberg Suggested here https://github.com/SeleniumHQ/selenium/issues/3651#issuecomment-291904277. It was producing wrong documentation because of aliasing.

Thanks

**Comment by Dude-X** [comment]
@igauravsehrawat Since proper documentation is more helpful than duplicate code, I'll see if I can get a core committer to make this change. Did you sign the CLA?

**Comment by igauravsehrawat** [comment]
@Dude-X Yes, I did sign the CLA.

Thanks a lot.

**Comment by lmtierney** [comment]
@igauravsehrawat can you throw a check in the checkbox for the CLA in your PR comment

File: py/selenium/webdriver/remote/webdriver.py [hunk 1]
```diff
@@ -837,7 +837,18 @@ def get_screenshot_as_file(self, filename):
...
+        Gets the screenshot of the current window. Returns False if there is
+           any IOError, else returns True. Use full paths in your filename.
+
+        :Args:
+         - filename: The full path you wish to save your screenshot to.
+
+        :Usage:
+            driver.save_screenshot('/Screenshots/foo.png')
+        """
+        return self.get_screenshot_as_file(self, filename)
```
Comment (lmtierney) [inline py/selenium/webdriver/remote/webdriver.py]: This needs to be just

    return self.get_screenshot_as_file(filename)

Comment (igauravsehrawat) [inline py/selenium/webdriver/remote/webdriver.py]: My bad, confused myself with explicit `self` logic.
Fixed it.

**Comment by lmtierney** [comment]
thanks @igauravsehrawat

**Comment by igauravsehrawat** [comment]
Thank you all for the help.

**Pull request merged**